lib/delayed/job_groups/backend/active_record/railtie.rb

```diff
@@ -0,0 +1,13 @@
+module Delayed
```

Comment:
It looks like this file is missing the `# frozen_string_literal: true` pragma. I've updated our CircleCI configuration to run builds on PRs from forks to catch this on your next commit. In the meantime you can run rubocop locally to detect and auto-correct the problem.